src/index.js

```diff
@@ -1,15 +1,30 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './pages/App';
 import Charcuterie from 'pages/Charcuterie';
 import reportWebVitals from './utils/reportWebVitals';
+import Button from 'components/Button'
 
 ReactDOM.render(
   <React.StrictMode>
-    {/* <App /> */}
-    <Charcuterie >
-      <p>Hi</p>
+    <Charcuterie>
+      <h3>small</h3>
+      <Button size={"small"} type={"primary"} label={"primary"} />
+      <Button size={"small"} type={"secondary"} label={"secondary"} />
+      <Button size={"small"} type={"text"} label={"text"} />
```

Comment:
can you verify the onClick functionality works? eg. add a Button like this: 
`<Button onClick={()=>{console.log("button clicked")}} />` 
and see if clicking it fires the console.log

Reply:
good catch, it wasn't able to handle any clicks before - add a clickable button to the page!